app-engine-daily-version-cleanup/main.tf

```diff
@@ -0,0 +1,68 @@
+data "google_compute_default_service_account" "default" {
+  project = var.project
+}
+
+resource "google_project_service" "required_api" {
+  for_each = toset([
+    "appengine.googleapis.com",
+    "cloudscheduler.googleapis.com"
+  ])
+  project = var.project
+  service = each.key
+}
+
+resource "google_cloudbuild_trigger" "tf-app-engine-clean-up-scheduled" {
+  project = var.project
+  name    = "tf-app-engine-clean-up-scheduled"
+
+  github {
+    owner = var.github_owner
+    name  = var.github_name
+    push {
+      branch = var.branches_to_apply_regex
+    }
+  }
+  disabled = true
+
+  build {
+    step {
+      name       = "gcr.io/cloud-builders/gcloud"
+      entrypoint = "bash"
+      args = ["-c",
+      "git clone https://github.com/aliz-ai/terraform-aliz-modules.git"]
+    }
+    step {
+      name       = "gcr.io/cloud-builders/gcloud"
+      entrypoint = "bash"
+      dir        = "terraform-aliz-modules"
+      args       = ["-c", 
+      "git checkout app-engine-daily-version-cleanup"]
+    }
```

Comment:
this step was needed because the module wasn't merged, right?
IF we merge AND delete the branch, this could break the pipeline

Reply:
I think you are right. Maybe the branch name can be a parameter with the default value to master.